pom.xml

```diff
@@ -175,4 +190,16 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-</project>  
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jenkins-ci.tools</groupId>
+                <artifactId>maven-hpi-plugin</artifactId>
+                <configuration>
+                    <compatibleSinceVersion>2.0</compatibleSinceVersion>
```

Comment:
Hmm, that is generally true of _any_ plugin release. Certainly we do not routinely add this attribute. It adds a warning to the update center which will be alarming.